source/extensions/filters/http/fault/fault_filter.cc

```diff
@@ -36,10 +38,6 @@ FaultFilterConfig::FaultFilterConfig(const envoy::config::filter::http::fault::v
     : runtime_(runtime), stats_(generateStats(stats_prefix, scope)), stats_prefix_(stats_prefix),
       scope_(scope) {
 
-  if (!fault.has_abort() && !fault.has_delay()) {
```

Comment:
Why was this removed?

Reply:
This is to support a fully empty fault filter config -- which essentially should do nothing. This will allow keeping the fault filter always on and shipping filter configs via RDS

Reply:
OK thanks